lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -118,6 +127,43 @@ $-visible-item-count: 10;
   }
 }
 
+.nx-transfer-list__item--with-reordering {
+  .nx-transfer-list__select {
+    border-radius: var(--nx-border-radius);
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+  }
+}
+
+.nx-transfer-list__button-bar {
+  border: var(--nx-border-default);
+  border-left: 0;
+  border-radius: var(--nx-border-radius);
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+  gap: var(--nx-spacing-base);
+  margin: 0;
+  padding: var(--nx-spacing-base) var(--nx-spacing-2x);
+}
+
+$-button-size: 32px;
+
+.nx-transfer-list__button {
+  height: $-button-size;
+  padding: 0;
+  margin: 0;
+  width: $-button-size;
```

Comment:
These are supposed to be our usual small-style icon-only-buttons right? The right way to make that is with the `small-icon-btn` mixin in `_nx-button-helpers.scss`

Reply:
> These are supposed to be our usual small-style icon-only-buttons right? The right way to make that is with the `small-icon-btn` mixin in `_nx-button-helpers.scss`

I gave that mixin a try but the dimension is different from the spec and adds an additional few pixels to the item height. 🤔 

<img width="911" alt="Screen Shot 2021-09-24 at 12 29 30 PM" src="https://user-images.githubusercontent.com/10160030/134709683-ae660db5-dbc9-4af1-ac2a-ef397e22b3d9.png">

<img width="243" alt="Screen Shot 2021-09-24 at 12 32 38 PM" src="https://user-images.githubusercontent.com/10160030/134709846-6497789e-e718-4c3d-b59f-af9e18e8a76c.png">

Reply:
> I gave that mixin a try but the dimension is different from the spec and adds an additional few pixels to the item height.

I doubt it was deliberate that these buttons be smaller than all of our other small icon-only buttons.  You should check with @alexalyons126 about that.  For reference, generally our small icon-only buttons are 34x34 including the border.  Looks like this mockup has them at 32x32.